xapian-core/tests/harness/backendmanager.cc

```diff
@@ -219,13 +233,25 @@ BackendManager::get_compaction_output_path(const std::string&)
     invalid_operation("Compaction not supported for this database type");
 }
 
+Xapian::WritableDatabase
+BackendManager::get_generated_database(const std::string& name)
+{
+    return get_writable_database(name, string());
+}
+
 string
 BackendManager::get_generated_database_path(const std::string &)
 {
     invalid_operation("Generated databases aren't supported for this database "
 		      "type");
 }
 
+void
+BackendManager::finalise_generated_database(const std::string&)
+{
+
+}
```

Comment:
Xapian coding style wouldn't put a blank line in an empty function like this - it doesn't add anything useful.